site/examples/js/inlines.jsx

```diff
@@ -370,8 +367,8 @@ const ToggleEditableButtonButton = () => {
   return (
     <Button
       active
-      onMouseDown={event => {
-        event.preventDefault()
+      onPointerDown={event => event.preventDefault()}
+      onClick={() => {
```

Comment:
these were changed by `yarn tsc:examples`, they already existed in the tsx files and weren't synced